google/resumable_media/requests/download.py

```diff
@@ -18,6 +18,8 @@
 import hashlib
 import logging
 
+import urllib3.response
```

Comment:
@jonparrott This makes it impossible to import this module unless `urllib3` is installed. Previously, `requests` could be uninstalled and everything would work perfectly if the passed in objects matched the expected interfaces.

I can move back to that approach and ditch the subclass. It's a [small implementation][1]:

```python
class GzipDecoder(object):

    def __init__(self):
        self._obj = zlib.decompressobj(16 + zlib.MAX_WBITS)

    def __getattr__(self, name):
        return getattr(self._obj, name)

    def decompress(self, data):
        if not data:
            return data
        return self._obj.decompress(data)
```

[1]: https://github.com/shazow/urllib3/blob/ff5c6749c237ff6a8b0a3450713ecce7469a7ea8/urllib3/response.py#L55-L66